services/order.py

```diff
@@ -0,0 +1,38 @@
+from django.db import transaction
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    user = User.objects.get(username=username)
+
+    with transaction.atomic():
```

Comment:
You should use transaction.atomic as decorator instead of context manager. It makes your code more readable